src/os/CMakeLists.txt

```diff
@@ -114,17 +120,20 @@ if(HAVE_LIBAIO)
 endif()
 
 if(WITH_SPDK)
+  include(Builddpdk)
   include_directories("${CMAKE_SOURCE_DIR}/src/spdk/include")
```

Comment:
please use `${DPDK_INCLUDE_DIR}`

Reply:
ok

Reply:
seems they are different: DPDK_INCLUDE_DIR means src/spdk/dpdk/build/include. In this please, we want to include spdk header files, not dpdk.

Reply:
ahh, right!